src/auth/cephx/CephxSessionHandler.cc

```diff
@@ -207,7 +207,8 @@ int CephxSessionHandler::sign_bufferlist(bufferlist &in, bufferlist &out)
 int CephxSessionHandler::encrypt_bufferlist(bufferlist &in, bufferlist &out) {
   std::string error;
   try {
-    connection_secret.encrypt(cct, in, out, &error);
+#warning fixme key
```

Comment:
? fixme

Reply:
this is all changing in @rzarzynski 's follow-on work